base/pkg/generate.jl

```diff
@@ -149,6 +150,17 @@ function travis(pkg::AbstractString; force::Bool=false)
         #script:
```

Comment:
That sounds better -- I reverted the script comment change so it again ends with `coverage=true)` in anticipation of the change. Thanks for the suggestions.